pom.xml

```diff
@@ -74,6 +74,7 @@
         <cxf.version>3.1.11</cxf.version>
         <bouncy.version>1.55</bouncy.version>
         <xmlsec.version>2.0.8</xmlsec.version>
+        <jts.version>1.12</jts.version>
```

Comment:
We pull in JTS 1.13, but our instructions for external solr deployments is to pull in 1.12, and we use 1.1 throughout the rest of our build, thirdparty bundling.  This includes where it was defined as 1.12 in catalog/pom.xml.  I moved this back to 1.12 here to match what we use when deploying solr externally, as well as match what we use in the catalog libraries.  If we want to have 1.13 here, we probably should be doing the same in catalog, and may want to consider upgrading to 1.14